components/provider/Cargo.toml

```diff
@@ -30,7 +30,7 @@ serialize_none = []
 hello_world = ["icu_locid_macros"]
 
 [dependencies]
-icu_locid = { version = "0.1", path = "../locid" }
+icu_locid = { version = "0.1", path = "../locid", features = ["serde"] }
```

Comment:
Can you add a comment saying that you need that feature for likelysubtags so that when I do #415 I'll remember to update it again?